tests/test_app/App/BakeApplication.php

```diff
@@ -0,0 +1,35 @@
+<?php
+declare(strict_types=1);
+
+namespace TestApp;
+
+use Cake\Console\CommandCollection;
+use Cake\Http\BaseApplication;
+use Cake\Http\MiddlewareQueue;
+use Cake\Routing\RouteBuilder;
+use Migrations\Test\Command\CustomBakeMigrationDiffCommand;
+
+class BakeApplication extends BaseApplication
```

Comment:
Application should suffice here IMO.

Reply:
`extends Application` ??

Reply:
class Application extends BaseApplication
see my plugins for example.

Reply:
there is a class as `Application` 
i can not use same name

Reply:
You are in TestApp
There are multiple ones in there already?

Reply:
yes

Reply:
merged with `Application`